tests/introspection/ApiCtorInitTest.cs

```diff
@@ -132,6 +132,9 @@ protected virtual bool Skip (Type type)
 				return true;
 			case "QTMovie":
 				return TestRuntime.CheckSystemVersion (PlatformName.MacOSX, 10, 14, 4); // Broke in macOS 10.14.4.
+			case "MTLFunctionConstantValues":
+				// the type was added with Xcode 8, but the default ctor in Xcode 9.
+				return !TestRuntime.CheckXcodeVersion (9, 0);
```

Comment:
it would be better to redefine the `.ctor` in the bindings and add the availability attribute